dv-export/python/dv_export_model.py

```diff
@@ -0,0 +1,98 @@
+from dataclasses import dataclass
+from enum import Enum
+
+
+class DVExportDataType(Enum):
+    String = 1
+    Integer = 2
+    Number = 3
+    Date = 4
+    Boolean = 5
+
+
+def string_to_data_type(string_value: str) -> DVExportDataType:
+    try:
+        return DVExportDataType[string_value]
+    except KeyError:
+        raise ValueError(f"No such DataType: {string_value}")
+
+
+@dataclass
+class FileInfo:
+    file_id: int
+    name: str
+
+
+@dataclass
+class ColumnInfo:
+    name: str
+    data_type: DVExportDataType
+    allows_null: bool
+    primary_key: bool
+
+
+@dataclass
+class ColumnInfoAndFileInfo:
+    column_infos: list[ColumnInfo]
+    file_infos: list[FileInfo]
+
+
+@dataclass
+class ColumnsMetadata:
+    common_columns: ColumnInfoAndFileInfo
+    new_columns: ColumnInfoAndFileInfo
+    deleted_columns: list[str]
+
+
+@dataclass
+class TableInfo:
+    name: str
+    columns: ColumnsMetadata
+
+
+def extract_tables(tables_metadata: list[dict[str, any]]) -> list[TableInfo]:
```

Comment:
As public sample code we should be a bit more diligent adding docstrings to non-trivial functions